File: clients/java/src/com/codeumentus/codeflow/CodeflowClientTest.java
package com.codeumentus.codeflow;

import java.util.HashMap;
import java.util.Map;

public class CodeflowClientTest {

	public static void main(String[] args) {
		CodeflowClient cfc = new CodeflowClient();
		cfc.setCodeflowUrl("http://localhost:8081/log");
		Double procid = (Math.random() * 10000);
		Map<String, String> keys = new HashMap();
		keys.put("testkey1", "testvalue1");
		keys.put("testkey2", "testvalue2");
		cfc.sendTrigger("customer-flow", "check-form", String.valueOf(procid.intValue()), "test", keys);
	}

}

File: clients/scala/src/main/scala/com/codeumentus/codeflow/CodeflowClientTest.scala
package com.codeumentus.codeflow


object CodeflowClientTest {

  def main(args: Array[String]): Unit = {
    var cfc = new CodeflowClient();
    cfc.codeflowUrl = "http://localhost:8081/log";
    val r = scala.util.Random;
    val processid = r.nextInt(100000);
    val keys = Map("testkey1" -> "testvalue1", "testkey2" -> "testvalue2")
    cfc.sendTrigger("customer-flow", "check-form", processid.toString, "test", keys);
  }

}
